doc/md/serving/intro.md

```diff
@@ -0,0 +1,38 @@
+---
+id: atlas-serving
+slug: /serving/intro
+title: Serving
+---
+
+## Introduction
+
+Usually as a team using Atlas, you would like to deploy it as a service in your cloud data center and enjoy a centralized 
+instance.
+When using Atlas in such cases we recommend using persistent storage.
```

Comment:
```suggestion
The Atlas Management UI is designed to be a long-running process. It is therefore recommended to run it against a persistent database that will make sure your team's schemas, database credentials, migration history and more are not lost in case you need to restart the service.
```

Reply:
Added